shapely/tests/test_ragged_array.py

```diff
@@ -586,3 +586,28 @@ def test_from_ragged_wrong_offsets():
             np.array([[0, 0], [0, 1]]),
             offsets=(np.array([0, 1]),),
         )
+
+
+def test_from_ragged_crash_2284():
+    # caused segfault in shapely 2.1.0
+    # https://github.com/shapely/shapely/discussions/2284
+
+    # one of the geometries has more rings than the total number of geometries
+    coords = np.random.randn(60, 2)
```

Comment:
New code should use the modern RNG methods ([docs](https://numpy.org/doc/2.2/reference/random/generated/numpy.random.randn.html)), also make the random coords stay the same-ish by specifying a seed.
```suggestion
    rng = np.random.default_rng(2284)
    coords = rng.standard_normal(120).reshape((60, 2))
```

Reply:
> also make the random coords stay the same-ish by specifying a seed.

For testing we don't necessarily want to use a fixed seed? It seems useful to me to use varying values. Of course if you then run into an issue with certain values, it is not reproducible, so it is not necessarily very helpful, but personally I don't see the point of wanting to fix the random values for this use case

Reply:
It probably doesn't make any difference on the randomness, as it's the data size that matters here. A non-seeded RNG is simply `rng = np.random.default_rng()`, if that's preferred.

Reply:
Updated. 
(now, to be honest, I personally think this recommendation from numpy is too strong, and for testing purposes here where we don't care about the pseudo-random number generator theory (we are not doing statistical modelling), using `np.random.default_rng().standard_normal(120).reshape((60, 2))` instead of `np.random.randn(60, 2)`  is just unnecessary boilerplate and noise in the test code .. ;))